thunder/dynamo/benchmark_utils.py

```diff
@@ -97,7 +97,7 @@ class TorchEagerSpecification(CompileSpecificationInterface):
     def __init__(self, specification_name="torcheager"):
         self.name = specification_name
 
-    def compile(self, fn):
+    def compile(self, fn, *args):
```

Comment:
I would suggest **kwargs instead of *args and then have the compile methods be:

```
def compile(self, fn, **kwargs):
```

and the `compile` method for `torch_inductor` would be:

```
def compile(self, fn, inputs, **kwargs)
```

or 

```
def compile(self, fn, *, inputs, **kwargs)
```

depending on whether you want to require that `inputs` be specified as a kwarg or not